src/app.d.ts

```diff
@@ -175,16 +175,26 @@ declare global {
         uid: string;
       }
 
+      interface InviteRequest {
+        timestamp: number;
+        message?: string;
+      }
+
       interface Public {
         name: string;
         contactEmail?: string;
       }
+      // Private fields can be seen and edited by org admins
       interface Private {
         members?: {
           [uid: string]: Member;
         };
         invites?: string[];
+        inviteRequests?: {
```

Comment:
New field in the db under `organizations/<org_id>/private`. I chose not to make a new unique resource for invites, I think all the info we need can be easily encapsulated within a couple of fields, and I didn't see any harm in nesting it into the org.